oem/zt/zt.go

```diff
@@ -0,0 +1,95 @@
+//
+// SPDX-License-Identifier: BSD-3-Clause
+//
+
+package zt
+
+import (
+	"encoding/json"
+	"fmt"
+	"net/http"
+
+	"github.com/stmcginnis/gofish/common"
+)
+
+var (
+	SubmitTestEventTarget = "/redfish/v1/EventService/Actions/EventService.SendTestEvent"
+)
+
+type SubscriptionZtRequestType struct {
+	Destination string            `json:"Destination"`
+	HTTPHeaders map[string]string `json:"HttpHeaders,omitempty"`
+	Protocol    string            `json:"Protocol,omitempty"`
+	Context     string            `json:"Context,omitempty"`
+}
+
+// subscribeResponseType zt uses a unique subscription response.
+type subscribeZtResponseType struct {
+	ODataContext         string `json:"@odata.context"`
+	ODataEtag            string `json:"@odata.etag"`
+	ODataID              string `json:"@odata.id"`
+	ODataType            string `json:"@odata.type"`
+	Context              string `json:"Context"`
+	DeliveryRetryPolicy  string `json:"DeliveryRetryPolicy"`
+	Description          string `json:"Description"`
+	Destination          string `json:"Destination"`
+	EventFormatType      string `json:"EventFormatType"`
+	ID                   int    `json:"ID"`
+	Name                 string `json:"Name"`
+	Protocol             string `json:"Protocol"`
+	common.Status        `json:"Status"`
```

Comment:
This doesn't look right. Defined this way, the properties of the `Status` struct are direct properties of the response type. But according to the JSON in the test code, there should be a `Status` property of type `common.Status`.

So current definition is:
```json
{
    "Health": "OK",
    "HealthRollup": "OK",
    "State": "Enabled"
}
```

But the example from the test JSON is:

```json
{
    "Status": {
        "Health": "OK",
        "HealthRollup": "OK",
        "State": "Enabled"
    }
}
```

Reply:
done